src/main/java/duke/Parser.java

```diff
@@ -0,0 +1,95 @@
+package duke;
+
+import duke.data.Storage;
+import duke.exception.*;
+
+import java.util.ArrayList;
+
+public class Parser {
```

Comment:
Perhaps, you could include a short comment on each of the classes to give the "secondary coder" a brief overall idea on what that specific class does